server/middleware/validation.js

```diff
@@ -29,6 +34,33 @@ const validators = {
         }
         return null;
     },
+    isValidDay({ label, value, message }) {
+        if (value) {
+            if (getDay(value) > new Date(getYear(value), getMonth(value), 0).getDate() || getDay(value) < 1) {
```

Comment:
This will still accept `0001` as a date -- which is fine, it should -- but it should ideally transform silently to `01`. Might be another ticket though?

Reply:
Going to leave this for now, can raise a ticket if we want to later